packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/_tests/RestaurantDish.test.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-unused-vars */
+import { mount } from '@vue/test-utils';
+import sut from '../RestaurantDish.vue';
+
```

Comment:
What does `sut` mean/short for?

Reply:
system under test - it's a relatively common shorthand for 'the thing being test' in unit tests

Reply:
Cool 👍